src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -99,7 +99,7 @@ public class DrsService {
   private static final String ACCESS_ID_SEPARATOR = "*";
   private static final String DRS_OBJECT_VERSION = "0";
   @VisibleForTesting static final Duration URL_TTL = Duration.ofMinutes(15);
-
+  static final Duration AZURE_URL_TTL = Duration.ofMinutes(60);
```

Comment:
Minor suggestion: should we pull this out into an environment variable that can be changed? It may not be worth the effort.

Reply:
Good idea - but, given this is for Azure, I don't think it is worth the effort.